content/html/concepts/tables/tables.md

```diff
@@ -93,11 +90,7 @@ The table data element, denoted as `<td>`, is used to display individual data wi
   </tr>
 </table>
 ```
-
-| Heading 1 | Heading 2 | Heading 3 |
-| :-------: | :-------: | :-------: |
-|  Cell 1   |  Cell 2   |  Cell 3   |
-|  Cell 4   |  Cell 5   |  Cell 6   |
+![image](https://github.com/Moddinu/docs/assets/5288834/2b15a2d2-d047-4426-a3bb-f04cfb137278)
```

Comment:
This image is in the wrong place. it needs to be uploaded to the `/media` folder and use the url:`https://raw.githubusercontent.com/Codecademy/docs/main/media/`